xain_fl/fl/coordinator/aggregate.py

```diff
@@ -99,44 +85,56 @@ def federated_averaging(thetas: List[Theta], weighting: np.ndarray) -> Theta:
     return theta_avg
 
 
-def evo_agg(thetas: List[Theta], evaluator: Evaluator) -> Theta:
-    """Experimental
+# TODO: decide how to continue with that
```

Comment:
Please create a ticket for it and put a ticket number into todo

Reply:
Done.